### ISSUE TITLE
Illegal value: strikethrough

### ISSUE THREAD

**Issue by vogler** [neoclide/coc.nvim#3134]
## versions

vim version: NVIM v0.3.4
node version: v10.24.0
coc.nvim version: 0.0.80-69e029ed9e
coc.nvim directory: /home/pi/.vim/bundle/coc.nvim
term: screen-256color
platform: linux

## Log of coc.nvim

2021-06-06T13:12:16.754 WARN (pid:17685) [attach] - Plugin not ready when received "highlight" []
2021-06-06T13:12:17.061 INFO (pid:17685) [plugin] - coc.nvim 0.0.80-69e029ed9e initialized with node: v10.24.0 after 259ms
2021-06-06T13:12:19.063 INFO (pid:17685) [attach] - receive notification: highlight []
2021-06-06T13:12:25.205 INFO (pid:17685) [attach] - receive notification: showInfo []

## Describe the bug

On start with coc.nvim enabled it complains:
~~~
Error detected while processing function <SNR>61_Hi:
line   10:
E418: Illegal value: strikethrough
Press ENTER or type command to continue
~~~

It disappears after I comment out
https://github.com/neoclide/coc.nvim/blob/06d950e547e8fa1a775399ae5eeb70603084b109/plugin/coc.vim#L353

Similar problem as #1430.
Haven't tested it with nvim 0.4.0.
It's running on a RPi with Debian 10 - updating to nvim 0.4.4 from testing resulted in conflicts.

## Reproduce the bug

- Create file `mini.vim` with：

  ```vim
  set nocompatible
  set runtimepath^=/path/to/coc.nvim
  filetype plugin indent on
  syntax on
  set hidden
  ```

- Start (neo)vim with command: `vim -u mini.vim`

-> Got the same message.

**Comment by Starslayerx** [comment]
Same issue after updated all my neovim plugs:
```bash
Error detected while processing function <SNR>19_Hi:
line   10:
E418: Illegal value: strikethrough
Press ENTER or type command to continue
```

**Comment by chemzqm** [comment]
Fixed by 509209a
